CONTRIBUTING.md

```diff
@@ -59,9 +59,9 @@ In this scenario, it’s possible that you found a bug or enhancement that we ha
 ## Working on Expensify Jobs
 *Reminder: For technical guidance please refer to the [README](https://github.com/Expensify/App/blob/main/README.md)*.
 
-#### Express interest for the job on Upwork.com
+#### Check GitHub for existing proposals from other users
 
-1. If you are interested in working on a job posted in Upwork, click **Submit a Proposal** in Upwork to express your interest to the Expensify team.
+1. If someone already posted a good solution, most probably they will be assigned the issue. You can still post proposals, but you should know that the best proposal that addresses the problem will be selected, all things equal we prioritize the first proposal.
```

Comment:
I think we could make this a little bit more clear with re-phrasing? I find `most probably they will be assigned the issue` to be a little unclear still. What if we re-structure like this?

>Expensify reviews all solution proposals on a first come first serve basis. If you see other contributors have already proposed a solution, you can still provide a solution proposal and we will review it if we don't select the first-received proposal(s). We look for the _earliest provided_ and _best_ proposed solution that addresses the job.

Reply:
Take what you like and leave what you don't like, just suggesting this for more clarity really.

Reply:
ok took most of it. Check it out

Reply:
Looking

Reply:
Looks great. Can I click [Resolve conversation] on this?